doc/user-guide/advanced/special-features.rst

```diff
@@ -151,8 +94,36 @@ changed the rupture classes or the surface classes, but we changed the seed algo
 different numbers to be generated (hopefully, statistically consistent). A bug fix or change of logic in the calculator 
 can also change the numbers across engine versions.
 
-``max_sites_disagg``
---------------------
+The ``custom_site_id``
```

Comment:
`custom_site_id` should probably go in the regular manual rather than under advanced features IMO

Reply:
Seconded, but not blocking the release